plugins/out_kinesis_streams/kinesis_api.c

```diff
@@ -295,11 +306,9 @@ static int process_event(struct flb_kinesis *ctx, struct flush *buf,
         time_key_ptr += 3;
         tmp_size = buf->tmp_buf_size - buf->tmp_buf_offset;
         tmp_size -= (time_key_ptr - tmp_buf_ptr);
-        len = strftime(time_key_ptr, tmp_size, ctx->time_key_format, &time_stamp);
-        if (len <= 0) {
-            /* ran out of space - should not happen because of check above */
-            return 1;
-        }
+        /* merge out_buf to time_key_ptr */
```

Comment:
Could you add a new line above the `        /* merge out_buf to time_key_ptr */` comment